### PR TITLE
chore: docs dir

### PR DESCRIPTION
**Pull request by Malix-Labs**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The following are not in scope:
 - Remove SUID-root from [numerous binaries](https://github.com/secureblue/secureblue/blob/live/files/scripts/removesuid.sh) and replace functionality [using capabilities](https://github.com/secureblue/secureblue/blob/live/files/system/usr/bin/setcapsforunsuidbinaries)
```
Comment (RoyalOughtness) [inline docs/README.md:32]: why is the readme also getting moved to the docs dir?

Comment (Malix-Labs) [reply docs/README.md:32]: README works in `/`, `docs/`, and `.github/` too

Comment (RoyalOughtness) [reply docs/README.md:32]: @Malix-Labs oh sweet, didn't know

**Review by RoyalOughtness — APPROVED** [review]

**Comment by Malix-Labs** [comment]
Also, maybe a [`docs/SECURITY.md`](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) file would be interesting to feat

It is about security policies, which you might easily be way more familiar than me with

**Comment by RoyalOughtness** [comment]
> Also, maybe a [`docs/SECURITY.md`](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) file would be interesting to feat
> 
> It is about security policies, which you might easily be way more familiar than me with

Yup I have a task for it, just need to get around to it. Probably after i get openscap scans working

https://github.com/secureblue/secureblue/issues/521

**Comment by Malix-Labs** [comment]
Ohkay, 2 days late lmao

**Review by EsseLowNitro — APPROVED** [review]

**Pull request merged**